docs/deployment.md

```diff
@@ -24,3 +24,49 @@ All PRs merged will be continuously deployed to `Production` using the below wor
 - You can view the current deployed commit in each of the environments
 
 <a href="https://github.com/DFE-Digital/find-teacher-training/deployments">![Environments](./deployments.png "GitHub Actions Workflow")</a>
+
+## On demand deployments
+
+### Using the Manual Deploy Action
+Any previously built commit sha can be deployed to any of the three environments from with the Actions page in [GitHub](https://github.com/DFE-Digital/find-teacher-training/actions?query=workflow%3ADeploy).
+
+<a href="https://github.com/DFE-Digital/find-teacher-training/actions?query=workflow%3ADeploy">![Deploy](./manual_deploy.png "Deploy Workflow")</a>
+
+
+### From local dev console
+Please follow the below instructions to deploy to any of the environments from your local console.
+
+Setup the following environment variables. Hint: Use [direnv](https://direnv.net) to load environments variables depending on the curent directory.
+```yml
+  ARM_ACCESS_KEY= # The terraform storage account access key from the Azure portal for the respective environment. 
```

Comment:
We tested it works if you login with `az` command line instead.
And it's better because it protects test and prod subscriptions with PIM